File: app/src/main/java/com/coderpig/drysisters/DrySisterApp.java
package com.coderpig.drysisters;

import android.app.Application;

/**
 * 描述：Application类
 *
 * @author coder-pig： 2016/08/07 15:07
 */
public class DrySisterApp extends Application {

    private static DrySisterApp context;

    @Override
    public void onCreate() {
        super.onCreate();
        context = this;
        DryInit.initTimber();
        DryInit.initOKHttp(this);
    }

    public static DrySisterApp getContext() {
        return context;
    }
}

File: app/src/main/java/com/coderpig/drysisters/DrySisterApp.kt
package com.coderpig.drysisters

import android.app.Application
import cn.coderpig.cp_base.CpKit

/**
 * Author: zpj
 * Date: 2023-09-25 16:06
 * Desc: 自定义Application类
 */
class DrySisterApp : Application() {
    companion object {
        var instance: Application? = null
    }

    override fun onCreate() {
        super.onCreate()
        instance = this
        CpKit.init(this)
        DryInit.initTimber()
        DryInit.initOKHttp(this)
    }
}